apps/cockroachdb/workload/test.yml

```diff
@@ -0,0 +1,79 @@
+---
+# This playbook generates the load on the cockroachdb application already deployed on the cluster.
+# Steps:
+# 1) Create litmus-result customr resource for logging.
+# 2) Create cockroachdb loadgen job
+# 3) Verify if the corresponding pod is running successfully.
+
+- hosts: localhost
+  connection: local
+
+  vars_files: 
+    - test_vars.yml
+
+  tasks:
+
+    - block:
+
+                ## RECORD START-OF-TEST IN LITMUS RESULT CR
+        - name: Generate the litmus result CR to reflect SOT (Start of Test)
+          template:
+            src: /litmus-result.j2
+            dest: litmus-result.yaml
+          vars:
+            test: "{{ test_name }}"
+            app: ""
+            chaostype: ""
+            phase: in-progress
+            verdict: none
+
+        - name: Apply the litmus result CR
+          shell: kubectl apply -f litmus-result.yaml
+          args:
+            executable: /bin/bash
+          register: lr_status
+          failed_when: "lr_status.rc != 0"
+
+        - name: Creating cockroachdb loadgen job
+          shell: kubectl apply -f {{ cockroachdb_loadgen }} -n {{ namespace }}
+          args:
+            executable: /bin/bash
+          register: result
+          failed_when: 'result.rc != 0'
+
+        - name: Verifying whether the cockroachdb loadgen pod is running.
```

Comment:
Would be great to have a check that loadgen has 'really' written data

Reply:
Thanks @ksatchit . This creates a database named 'test' with table 'kv' loaded with randomly generated key,values. Now, checking if such table resides inside pod.